pkg/handler/handler.go

```diff
@@ -149,6 +149,7 @@ func (o *OpenAPIService) RegisterOpenAPIVersionedService(servePath string, handl
 	}{
 		{"application", "json", o.getSwaggerBytes},
 		{"application", "com.github.proto-openapi.spec.v2@v1.0+protobuf", o.getSwaggerPbBytes},
+		{"application", "com.github.proto-openapi.spec.v2.v1.0+protobuf", o.getSwaggerPbBytes},
```

Comment:
I'll let @liggitt comment on this, I don't have a strong opinion, not sure how we built this in the first place or if the standard defines anything?

Reply:
replaced subtypes with constants